packages/core-data/src/fetch/__experimental-fetch-link-suggestions.ts

```diff
@@ -240,6 +246,34 @@ export default async function fetchLinkSuggestions(
 		);
 	}
 
+	if ( ! type || type === 'post-type-archive' ) {
+		queries.push(
+			apiFetch< PostTypesAPIResult[] >( {
+				path: addQueryArgs( '/wp/v2/types', {
+					page,
+					per_page: perPage,
```

Comment:
I don't believe that endpoint needs pagination.

Reply:
Yes in fact it might even be detrimental. This was my copy/paste mistake which I'll fix.

One thing I did wonder is if we could amend the endpoint to add support for querying for only Types which have an Archive and are "public".

Reply:
✅